components/file_preview/file_preview.test.tsx

```diff
@@ -69,6 +71,9 @@ describe('FilePreview', () => {
         const wrapper = shallow(
             <FilePreview {...props}/>
         );
+
+        // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+        //@ts-ignore
```

Comment:
This is a big code smell, what's the error being thrown here?
If it's around the `shallow()` method, you can use the generic type `ShallowWrapper` to define your component.

Reply:
the issue is that the handleRemove method does not exist on wrapper.instance(). I found a similar issue from 2018 and one suggestion from the post is to cast the instance.